.vsts-ci.yml

```diff
@@ -23,7 +23,7 @@ variables:
   - name: _DotNetValidationArtifactsCategory
     value: .NETCore
   - name: PostBuildSign
-    value: true
+    value: false
```

Comment:
@mmitche Should this change be merged into RC2, or did that just apply to the rc1-VSMac branch?

Reply:
Don't merge this change. I'll make the change separately in the rc2 branch